### PR TITLE
Add J4 requirements to the technical requirements page

### PR DESCRIPTION
**Pull request by wilsonge**

### CODE REVIEW

File: templates/joomla/html/com_content/article/requirements.php [hunk 1]
```diff
@@ -133,6 +133,80 @@
...
+                <td>2.0</td>
+                <td class="hidden-phone">
+                    <div><a href="https://www.apache.org" target="_blank" rel="noopener noreferrer">https://www.apache.org</a></div>
+                    <small><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_APACHE_REQUIREMENTS_FOOTNOTE'); ?></small>
+                </td>
+            </tr>
+            <tr>
+                <th scope="row">Nginx</th>
+                <td>1.18 +</td>
+                <td>1.0</td>
```
Comment (HLeithner) [inline templates/joomla/html/com_content/article/requirements.php:195]:
```suggestion
                <td>1.10</td>
```

File: templates/joomla/html/com_content/article/requirements.php [hunk 2]
```diff
@@ -133,6 +133,80 @@
...
+                <td class="hidden-phone">
+                    <a href="https://www.nginx.com/resources/wiki/" target="_blank" rel="noopener noreferrer">https://www.nginx.com/resources/wiki/</a>
+                </td>
+            </tr>
+            <tr>
+                <th scope="row">Microsoft IIS<sup><a href="#footnote-iis">[6]</a></sup></th>
+                <td>10 +</td>
+                <td>7</td>
```
Comment (HLeithner) [inline templates/joomla/html/com_content/article/requirements.php:203]:
```suggestion
                <td>8</td>
```

File: templates/joomla/html/com_content/article/requirements.php [hunk 3]
```diff
@@ -133,6 +133,80 @@
...
+            </tr>
+            </tbody>
+            <tbody>
+            <tr>
+                <th scope="rowgroup" colspan="4"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SUPPORTED_WEB_SERVERS'); ?></th>
+            </tr>
+            <tr>
+                <th scope="row">Apache<sup><a href="#footnote-apache">[3]</a></sup></th>
+                <td>2.4 +</td>
+                <td>2.0</td>
```
Comment (HLeithner) [inline templates/joomla/html/com_content/article/requirements.php:186]:
```suggestion
                <td>2.4</td>
```

File: templates/joomla/html/com_content/article/requirements.php [hunk 4]
```diff
@@ -133,6 +133,80 @@
...
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_RECOMMENDED'); ?></th>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MINIMUM'); ?></th>
+                <th class="hidden-phone" scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MORE_INFORMATION'); ?></th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <th scope="row">PHP</th>
+                <td>7.4</td>
+                <td>7.2</td>
```
Comment (HLeithner) [inline templates/joomla/html/com_content/article/requirements.php:151]:
```suggestion
                <td>7.2.5</td>
```

File: templates/joomla/html/com_content/article/requirements.php [hunk 5]
```diff
@@ -133,6 +133,80 @@
...
 
 		<h2><?php echo JText::_('TECHNICAL_REQUIREMENTS_HEADING_REQUIREMENTS_FOR_UNSUPPORTED_SOFTWARE'); ?></h2>
+        <h3><?php echo JText::_('TECHNICAL_REQUIREMENTS_HEADING_REQUIREMENTS_FOR_JOOMLA_4'); ?></h3>
+        <div class="alert alert-warning"><?php echo JText::_('TECHNICAL_REQUIREMENTS_ALERT_UNRELEASED_VERSION'); ?></div>
+        <table class="table table-bordered table-striped">
+            <thead>
+            <tr>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SOFTWARE'); ?></th>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_RECOMMENDED'); ?></th>
+                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MINIMUM'); ?></th>
```
Comment (HLeithner) [inline templates/joomla/html/com_content/article/requirements.php:143]:
```suggestion
                <th scope="col"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_MINIMUM'); ?><sup><a href="#footnote-minimum">[7]</a></th>
```

And add a footnote with some text that "Minimum requirements is the minimum supported version and versions may or may not work"

File: templates/joomla/html/com_content/article/requirements.php [hunk 6]
```diff
@@ -133,6 +133,80 @@
...
+                </td>
+            </tr>
+            </tbody>
+            <tbody>
+            <tr>
+                <th scope="rowgroup" colspan="4"><?php echo JText::_('TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SUPPORTED_DATABASES'); ?></th>
+            </tr>
+            <tr>
+                <th scope="row">MySQL</th>
+                <td>5.6 +</td>
```
Comment (HLeithner) [inline templates/joomla/html/com_content/article/requirements.php:163]:
```suggestion
                <td>5.7 +</td>
```
We shouldn't suggest EOL versions

**Review by HLeithner — CHANGES_REQUESTED** [review]

File: templates/joomla/html/com_content/article/requirements.php (hunk 6 above)
Comment (wilsonge) [reply templates/joomla/html/com_content/article/requirements.php:163]: This was the version approved by Production

File: templates/joomla/html/com_content/article/requirements.php (hunk 5 above)
Comment (wilsonge) [reply templates/joomla/html/com_content/article/requirements.php:143]: If it's a minimum supported verison we kinda guarentee it to work. Either way that's not j4 specific. That's what our recommended version is based around no?

Comment (HLeithner) [reply templates/joomla/html/com_content/article/requirements.php:143]: You remember the discussion about the motion text after it got public ? It was wrongly interpreted as minimum required version and not as minimum supported version. Which could increase our support load .

**Comment by conconnl** [comment]
@wilsonge & @HLeithner 
Can we process this PR, so the translators can start working on this?

**Comment by HLeithner** [comment]
Waiting for the requirement footnote changes by george

**Comment by wilsonge** [comment]
This looks ready now. @HLeithner can you take a final look please?

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**